lib/carin_for_blue_button_test_kit/generator/templates/suite.rb.erb

```diff
@@ -94,6 +95,9 @@ module CarinForBlueButtonTestKit
     <% group_id_list.each do |id| %>
         group from: :<%= id %><% end %>
       end
+<% if module_name == "CARIN4BBV200" %>      group from: :c4bb_server_v200_visual_inspection_and_attestation do
```

Comment:
In order to not have white space be added to the generate tests, you can do `<%- -%>`, but then add `trim_mode: '-'` to the output (line 25 in `suite_generator.rb`) (like I did on line 28 of `group_generator.rb`)

Reply:
thank you!!!